.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - "3.6"
+install:
+    - pip install -e .[test]
```

Comment:
I'd remove the `-e` for Travis testing - it should only be used for local development.